### PR TITLE
Update protomaps and refactor socrata to use geojsonmixin

### PR DESCRIPTION
**Pull request by nf-s**
### What this PR does

- `SocrataCatalogGroup` will not not return groups for Facets if there is only one - so it skips an unnecessary group level.
- Refactor `SocrataMapViewCatalogItem` to use `GeoJsonMixin`
  - So we can ideal zoom and other geojson mixin stuff
- Update protomaps.js to `1.5.0`
  - Fix to circle symoblizer

image

### Checklist

-   [x] ~There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)~ tests already exist
-   [x] ~I've updated relevant documentation in `doc/`.~ none needed
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/ModelMixins/GeojsonMixin.ts [hunk 1]
```diff
@@ -107,6 +107,18 @@ function GeoJsonMixin<T extends Constructor<Model<GeoJsonTraits>>>(Base: T) {
...
+    constructor(...args: any[]) {
+      super(...args);
+      if (this.strata.get(GeoJsonStratum.stratumName) === undefined) {
+        runInAction(() => {
+          this.strata.set(
+            GeoJsonStratum.stratumName,
+            GeoJsonStratum.load(this)
+          );
+        });
+      }
```
Comment (na9da) [inline lib/ModelMixins/GeojsonMixin.ts:119]: Any reason to move this to constructor from `forceLoadMetadata`?

**Review by na9da — APPROVED** [review]

File: lib/ModelMixins/GeojsonMixin.ts (hunk 1 above)
Comment (nf-s) [reply lib/ModelMixins/GeojsonMixin.ts:119]: I moved it because it doesn't load anything (there is no async call) - it is only used to set `rectangle`

Comment (nf-s) [reply lib/ModelMixins/GeojsonMixin.ts:119]: https://github.com/TerriaJS/terriajs/blob/90931527d585fda06e87152f94235045802259ff/lib/Core/AsyncLoader.ts#L9

Comment (nf-s) [reply lib/ModelMixins/GeojsonMixin.ts:119]: We should only use AsyncLoaders if there is an async call

**Pull request merged**